src/Foundation/Enum.cs

```diff
@@ -1332,4 +1332,9 @@ public enum NSUrlSessionDelayedRequestDisposition : long {
 		UseNewRequest = 1,
 		Cancel = 2,
 	}
+
+	[Native]
+	public enum NSXpcConnectionOptions : ulong {
+		Privileged = (1 << 12),
+	}
```

Comment:
Is this enum present in all platforms in the earliest version that we support?

Reply:
The enum is marked as `API_AVAILABLE(macos(10.8), ios(6.0), watchos(2.0), tvos(9.0))` in the SDK header. I see no reason why it wouldn't be.

Reply:
I needed the double check :)